.github/workflows/build-test-deploy.yml

```diff
@@ -15,104 +15,29 @@ permissions:
   pull-requests: write
 
 env:
-  # NOTE: keep in sync with docs
-  NIX_NIXPKGS_CHANNEL: https://nixos.org/channels/nixpkgs-22.05-darwin
-  NIX_CONTAINER_IMAGE: niteo/nixpkgs-nixos-22.05:316b762afdb9e142a803f29c49a88b4a47db80ee
-  NIX_INSTALL_SCRIPT: https://releases.nixos.org/nix/nix-2.10.3/install
-
-  CACHIX_COMPOSABLE: composable-community
+  NIX_VERSION: nix-2.13.2
+  NIXPKGS_CHANNEL: nixos-22.11
   CACHIX_COMPRESSION_LEVEL: 3
+  CACHIX_NAME: composable-community
 
   DOCKER_REGISTRY_NAME: composablefi
   # weird, next is implicit dependency of docker action
   DOCKER_USER_OPTION: '$UID:$GID'
 
 jobs:
-  check-privilege:
-    name: "Privilege Check"
+  privilege-check:
+    name: "Privilege check"
     if: ${{ github.event_name == 'push' || (github.event_name == 'pull_request_target' && github.event.pull_request.head.repo.id != 383289760) || (github.event_name == 'pull_request' && github.event.pull_request.head.repo.id == 383289760) }}
```

Comment:
Maybe while we are cleaning up, move 383289760 to an env variable

Reply:
Fixed in e2053b0f1feea96a4486328248b1e23594393a0d